src/tpt_reports/tpt_reports.py

```diff
@@ -60,7 +144,11 @@ def main() -> None:
                 error="Possible values for --log-level are "
                 + "debug, info, warning, error, and critical.",
             ),
+            "SERVICENOW_ID": Use(str, error="SERVICENOW_ID must be a string."),
+            "ELECTION_NAME": Use(str, error="ELECTION_NAME must be a string."),
+            "DOMAIN_TESTED": Use(str, error="DOMAIN_TESTED must be a string."),
             "JSON_FILE_PATH": Use(str, error="JSON_FILE_PATH must be an string."),
+            "OUTPUT_DIRECTORY": Use(str, error="OUTPUT_DIRECTORY must exist."),
```

Comment:
This error message is misleading because it indicates that you have checked if the output directory exists, but all this line is checking is whether or not `"OUTPUT_DIRECTORY"` is a string.

Since you later create the output directory if it doesn't already exist, you should modify the error message here to be more accurate:
```suggestion
            "OUTPUT_DIRECTORY": Use(str, error="OUTPUT_DIRECTORY must be a string."),
```